### PR TITLE
ci: distelli integration

### PR DESCRIPTION
**Pull request by captn3m0**

### CODE REVIEW

File: dockerconf/fastcgi.conf.j2 [hunk 1]
```diff
@@ -22,9 +22,6 @@ fastcgi_param  SERVER_PORT        $server_port;
 fastcgi_param  SERVER_NAME        $server_name;
 
 # TODO: Change these calls to lookup
-fastcgi_param  DB_HOST				{{ lookup('db_host') }}
-fastcgi_param  DB_DATABASE				{{ lookup('db_name') }}
-fastcgi_param  DB_USERNAME				{{ lookup('db_username') }}
```
Comment (captn3m0) [inline dockerconf/fastcgi.conf.j2]: DB_USERNAME doesn't have a default value: https://github.com/razorpay/concierge/blob/368c45a6555dbd5b9d1d5d80ce14c45bc0402a41/config/database.php#L55-L63

File: dockerconf/fastcgi.conf.j2 [hunk 2]
```diff
@@ -34,16 +31,14 @@ fastcgi_param  GOOGLE_CLIENT_SECRET				{{ lookup('gogle_client_secret') }}
...
-fastcgi_param  APP_DOMAIN				{{ lookup('app_domain') }}
+fastcgi_param  APP_DOMAIN				https://concierge.razorpay.com
 # something.vpc perhaps?
-fastcgi_param  BASE_URL				{{ lookup('base_url') }}
+fastcgi_param  BASE_URL				razorpay.com
 fastcgi_param  CRON_PASSWORD				{{ lookup('cron_password') }}
 
 # Use these if we don't have kube2iam working
 # AWS
-fastcgi_param  AWS_ACCESS_KEY_ID				{{ lookup('aws_access_key_id') }}
```
Comment (captn3m0) [inline dockerconf/fastcgi.conf.j2]: lgtm. tag once vishnu changes are filed?

File: dockerconf/fastcgi.conf.j2 (hunk 1 above)
Comment (tasdikrahman) [reply dockerconf/fastcgi.conf.j2]: @captn3m0 has been added https://github.com/razorpay/concierge/pull/18/commits/5acf6223eb318d69cf8f02daff473e034102df58

File: dockerconf/fastcgi.conf.j2 (hunk 2 above)
Comment (tasdikrahman) [reply dockerconf/fastcgi.conf.j2]: ^ cc @Hashfyre

**Pull request merged**